### PR TITLE
Create screens and components the edit profile & feed

### PR DESCRIPTION
**Pull request by madera-design**
# Description

What did you implemented/Why did you implemented this:

- Create components & screens feed view
- create components & screens edit profile view

image
image

### CODE REVIEW

File: ContratApp/src/components/CardItem/CardItem.tsx
Comment (DerekAyala) [inline ContratApp/src/components/CardItem/CardItem.tsx:1]: text1 , text2 and text3 are not very describing names.

**Review by DerekAyala — APPROVED** [review]
Nice Job guys, try to add node modules to gitignore.

**Review by FebernalGober — APPROVED** [review]

**Review by Antonio3veces3 — APPROVED** [review]
Wel done!

File: ContratApp/src/components/CardItem/CardItem.tsx
Comment (madera-design) [reply ContratApp/src/components/CardItem/CardItem.tsx:1]: Those names were for example, I have already given them the name that describes the value they will carry

**Pull request merged**